### DISCUSSION TITLE
Using an already existing table

### DISCUSSION THREAD

**Discussion by lukejcn** [eduardo-marcolino/acf-fields-in-custom-table#6] [Q&A] [answered]
Hello, this plugin looks great!
I’m currently using the geodirectory plugin, which already stores post meta in its own tables. Is it possible to use this plugin to edit those tables? It looks like this plugin adds a “ACF” prefix to the table?

**Comment by eduardo-marcolino** [comment]
Hello there. Yes, i'm adding a filter for changing the table prefix in the next update

**Comment by eduardo-marcolino** [comment, marked as answer]
Version 0.5 is out and we can now change table prefix as needed
